carbon_black_cloud/plugin.spec.yaml

```
@@ -0,0 +1,456 @@
+      required: false
+    scanStatus:
+      title: "Scan Status"
+      type: string
+      description: "Scan status"
+      required: false
+    sensorKitType:
+      title: "Sensor Kit Type"
+      type: string
+      description: "Sensor kit type"
+      required: false
+    sensorOutOfDate:
+      title: "Sensor Out of Date"
+      type: boolean
+      description: "Sensor out of date"
+      required: false
+    sensorPendingUpdate:
+      title: "Sensor Pending Update"
+      type: boolean
+      description: "Sensor pending update"
+      required: false
+    sensorStates:
+      title: "Sensor States"
+      type: "[]string"
+      description: "Sensor states"
+      required: false
+    sensorVersion:
+      title: "Sensor Version"
+      type: string
+      description: "Sensor version"
+      required: false
+    status:
+      title: "Status"
+      type: string
+      description: "Status"
+      required: false
+    targetPriorityType:
+      title: "Target Priority Type"
+      type: string
+      description: "Target priority type"
+      required: false
+    testId:
+      title: "Test ID"
+      type: integer
+      description: "Test ID"
+      required: false
+    uninstallCode:
+      title: "Uninstall Code"
+      type: string
+      description: "Uninstall code"
+      required: false
+    uninstalledTime:
+      title: "Uninstalled Time"
+      type: string
+      description: "Uninstalled time"
+      required: false
+    vdiBaseDevice:
+      title: "VDI Base Device"
+      type: string
+      description: "VDI Base Device"
+      required: false
+    virtualMachine:
+      title: "Virtual Machine"
+      type: boolean
+      description: "Virtual machine"
+      required: false
+    virtualizationProvider:
+      title: "Virtualization Provider"
+      type: string
+      description: "Virtualization Provider"
+      required: false
+    windowsPlatform:
+      title: "Windows Platform"
+      type: string
+      description: "Windows platform"
+      required: false
+
+connection:
+  api_id:
+    title: API ID
+    type: string
+    description: API ID
+    example: ABCDE12345
+    required: true
+    order: 1
+  api_secret_key:
+    title: API Secret Key
+    type: credential_secret_key
+    description: API Secret Key
+    example: ABCDEFG12345ABCD1234ABCD
+    required: true
+    order: 2
+  url:
+    title: URL
+    type: string
+    description: API URL
+    default: https://api5.conferdeploy.net/integrationServices/v3/
+    example: https://api5.conferdeploy.net/integrationServices/v3/
+    required: true
+    order: 3
+actions:
+  get_agent_details:
+    title: Get Agent Details
+    description: Get Agent Details
+    input:
+      agent:
+        title: Agent
+        description: Agent to retrieve device information from. Accepts IP address, MAC address, hostname, or device ID. Search results are case sensitive
+        type: string
+        required: true
+        example: 198.51.100.1
```

Minor nit, .1 is usually a gateway address. This is a better example per convention doc

```suggestion
        example: 198.51.100.100
```